### DISCUSSION TITLE
Clearing potoken and visitorData vlaues

### DISCUSSION THREAD

**Discussion by onexecute** [JuanBindez/pytubefix#294] [General]
Is there a way to clear potoken and visitorData values in order to re-enter them again?

**Comment by felipeucelli** [comment]
By default, pytubefix saves tokens in `pytubefix.__cache__`. You can also use:
```python 
YouTube(URL, allow_oauth_cache=False)
```
This will ask for token input on each run.

**Comment by onexecute** [comment]
Is it possible to make poToken work with playlists

```
from pytubefix import Playlist
from pytubefix.cli import on_progress

url = input("url here >")

pl = Playlist(url)

for video in pl.videos:
    ys = video.streams.get_highest_resolution()
    ys.download()
```

**Reply by felipeucelli** [reply]
Yes, it is possible to pass the potoken to the playlist, but keep in mind that it will only be used to build YouTube objects internally and not to call the playlist API (so far it is not necessary).

To pass the PoToken to a Playlist object, it's the same process as YouTube:

```python
Playlist(URL, use_po_token=True)
```

**Comment by onexecute** [comment]
Thank you!
